### PR TITLE
Add search to Clips4sale

### PR DESCRIPTION
**Pull request by JackDawson94**
Add search scenes by name to Clips4sale.

Haven't touched the existing scene scraper.

### CODE REVIEW

**Review by thismanyboyfriends — CHANGES_REQUESTED** [review]
I've tried this out myself and I've found that whilst the search works well, and I get a list of results (often correct/very accurate), if I try to actually scrape with them I get an unsupported protocol error.

Specific error message is:

`error while fragment scraping with scraper Clips4Sale: get (search text) unsupported protocol sceme ""`

A shame as I really want a clips4sale name scraper, there's a lot of content on there and manually searching URLs is tedious.

File: scrapers/Clips4Sale.yml [hunk 1]
```diff
@@ -10,7 +10,30 @@ sceneByURL:
...
+
 xPathScrapers:
+  c4sSearch:
+    common:
+      $card: //figure[contains(@class, "splide__slide")]
+    scene:
+      Title:
+        selector: $card/@aria-label
+      URL:
+        selector: $card//a[contains(@class, "search-clip__title")]
```
Comment (thismanyboyfriends) [inline scrapers/Clips4Sale.yml:30]: Found the error. This xpath resolves to the inner text, not the URL. You want to append `/@href` on the end.

Comment (JackDawson94) [reply scrapers/Clips4Sale.yml]: Thanks, pushed a fix :)

**Comment by scruffynerf** [comment]
works for me!

**Review by thismanyboyfriends — APPROVED** [review]

**Pull request merged**